src/components/ipAutozoomControl.ts

```diff
@@ -95,7 +89,7 @@ export default class IpAutozoomControl implements IAutozoomControl {
     const cnnFeature = new huddly.CnnFeature();
     cnnFeature.setFeature(feature);
     const azStatus = await this._deviceManager.getCnnFeatureStatus(cnnFeature);
-    return azStatus.getEnabled();
+    return azStatus.getAzStatus().getAzEnabled();
```

Comment:
Is this the weird case when you have a AzStatus inside CnnStatus or something?

Reply:
Yes, the only way it was done before. At some point I thought it was backwards compatible ☹️